packages/react-reconciler/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "react-reconciler",
+  "description": "React is a JavaScript library for building user interfaces.",
+  "version": "16.0.0-rc.2",
```

Comment:
Should we version this separately? Regular changes to React will be breaking changes for the reconciler.
We should either start with `0.0.1` or `1.0.0` 😄